libr/main/radiff2.c

```diff
@@ -1006,12 +1005,10 @@ R_API int r_main_radiff2(int argc, const char **argv) {
 		case 'h':
 			return show_help (1);
 		case 's':
-			if (mode == MODE_DIST) {
+			if (mode == MODE_DIST_MYERS) {
```

Comment:
this sounds wrong to me, why dist-myers should be enabling the dist-levenstein mode? and use myers as alternative? the namings are confusing.

Reply:
imho those MODEs should be defined in r_util  and use the same identifiers in `r_diff_buffers_distance` . Also your change is making `r_diff_buffers_distance_original` to be dead code. so there's no way to compare those myers/levenstein algorithms with the plain simple implementation (despite being slower)

Reply:
'l' is buggy (r_diff_buffers_distance_levenshtein) is buggy. I left a FIXME.